simavr/Makefile

```diff
@@ -43,7 +43,7 @@ sim_o 	:= ${patsubst sim/%.c, ${OBJ}/%.o, ${sim}}
 VPATH	= cores
 VPATH	+= sim
 
-IPATH	= sim
+IPATH	+= sim
```

Comment:
this bit is necessary to help gcc/clang find the includes to `libelf`. I tend to believe that on non-OSX systems, it should be harmless, as `IPATH` is anyway not yet setup at this stage.